hw1/task04/test_task04.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from task04 import check_sum_of_four
+
+all_zero_x5 = [0] * 5
+empty = []
+max_length = list(range(1000))
+
+
+@pytest.mark.parametrize(
+    ("value", "expected_result"),
+    [
+        ((all_zero_x5, all_zero_x5, all_zero_x5, all_zero_x5), 625),
+        ((empty, empty, empty, empty), 0),
+        ((max_length, max_length, max_length, max_length), 1),
```

Comment:
I'd say this test value does not bring anything useful for the test, let's remove it.

Reply:
Are you sure? Without this test quadruple for loop will do the job